### PR TITLE
Alternative documentation

### PR DESCRIPTION
**Pull request by georgeedwards**
Here is an updated ReadMe to fit in with my proposed Wiki Structure.

### CODE REVIEW

**Comment by ericmdantas** [comment]
Nice job, George. Thanks for the PR!

I'll merge it later today when I have time to take a closer look.

File: README.md [hunk 1]
```diff
@@ -1,70 +1,58 @@
...
-For a detailed comparison of the options and trade-offs between them, please visit [this](http://www.dancancro.com/comparison-of-angularjs-application-starters) link.
+    $ npm install -g generator-ng-fullstack
 
+then to create a new app:
 
-### inspiration
+    $ yo ng-fullstack
 
-Inspired by [generator-angular-fullstack](https://github.com/DaftMonk/generator-angular-fullstack).
+and answer the on-screen questions. When it's done, you should have [[this | Getting-Started#result]] structure. The default starter project is our [[Todo app | Todo-walkthrough]].
```
Comment (ericmdantas) [inline README.md]: links for the wiki are broken

File: README.md [hunk 2]
```diff
@@ -1,70 +1,58 @@
...
 
+For full guidance on setting up the dependencies and generating your first app, see [[Getting Started | getting-started]]
```
Comment (ericmdantas) [inline README.md]: also broken

**Comment by ericmdantas** [comment]
Just two minor comments and this is ready to merge.

Thanks once again.

**Comment by georgeedwards** [comment]
Sure, didn't spot the different syntax between the Wiki and Readme! Sorted now

**Comment by ericmdantas** [comment]
The getting started is still broken, but I'll fix it here.

Thanks!

**Pull request merged**